purchases/src/main/kotlin/com/revenuecat/purchases/common/Backend.kt

```diff
@@ -198,7 +198,7 @@ internal class Backend(
             "observer_mode" to observerMode,
             "price" to receiptInfo.price,
             "currency" to receiptInfo.currency,
-            "attributes" to subscriberAttributes.takeUnless { it.isEmpty() },
+            "attributes" to subscriberAttributes.takeUnless { it.isEmpty() || appConfig.customEntitlementComputation },
```

Comment:
we actually didn't have a test for this at all, so if you removed this line, tests would pass

Reply:
I added a test case for custom entitlements computation enabled and disabled

Reply:
LOOL nevermind there's an entire file dedicated to tests of exactly this. They're just not in this file and I hadn't searched thoroughly enough.